docs/content/get-started/configuration.md

```diff
@@ -162,21 +162,23 @@ If not specified, then the global defaults will be used.
 {{% /table %}}
 
 ### Pod Annotations
 
 {{% table %}}
 | Annotation                                                                                                                                                        | Values                                 | Default       |
 |-------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------|---------------|
-| [injector.nsm.nginx.com/auto-inject]({{< ref "/guides/inject-sidecar-proxy.md#automatic-proxy-injection" >}}) (deprecated)                                        | `true`, `false`                        | `true`        |
+| [injector.nsm.nginx.com/auto-inject]({{< ref "/guides/inject-sidecar-proxy.md#automatic-proxy-injection" >}}) (deprecated, use label instead)                     | `true`, `false`                        | `true`        |
 | [config.nsm.nginx.com/mtls-mode]({{< ref "/guides/secure-traffic-mtls.md#change-the-mtls-setting-for-a-resource" >}})                                             | `off`, `permissive`, `strict`          | `permissive`  |
 | [config.nsm.nginx.com/client-max-body-size](#client-max-body-size)                                                                                                | `0`, `64k`, `10m`, ...                 | `1m`          |
 | [config.nsm.nginx.com/ignore-incoming-ports]({{< ref "/guides/inject-sidecar-proxy.md#ignore-specific-ports" >}})                                                 | list of port strings                   | ""            |
 | [config.nsm.nginx.com/ignore-outgoing-ports]({{< ref "/guides/inject-sidecar-proxy.md#ignore-specific-ports" >}})                                                 | list of port strings                   | ""            |
 | [config.nsm.nginx.com/default-egress-allowed]({{< ref "/tutorials/kic/deploy-with-kic.md#enable-egress" >}})                                                    | `true`, `false`                        | `false`       |
-| [nsm.nginx.com/enable-ingress]({{< ref "/tutorials/kic/deploy-with-kic.md#secure-communication-between-nginx-plus-ingress-controller-and-nginx-service-mesh" >}}) | `true`, `false` | `false`       |
-| [nsm.nginx.com/enable-egress]({{< ref "/tutorials/kic/deploy-with-kic.md#enable-egress" >}})                                                                    | `true`, `false` | `false`       |
+| [nsm.nginx.com/enable-ingress]({{< ref "/tutorials/kic/deploy-with-kic.md#secure-communication-between-nginx-plus-ingress-controller-and-nginx-service-mesh" >}}) (deprecated, use label instead) | `true`, `false` | `false`       |
+| [nsm.nginx.com/enable-egress]({{< ref "/tutorials/kic/deploy-with-kic.md#enable-egress" >}}) (deprecated, use label instead)                                      | `true`, `false` | `false`       |
```

Comment:
Screenshot of these changes from local testing.
![Screen Shot 2022-12-30 at 11 41 25 AM](https://user-images.githubusercontent.com/50845045/210107301-a6d6ffe4-382c-46d3-a556-49c576873586.png)